### PR TITLE
home task 1

### PR DESCRIPTION
**Pull request by Tretykhlib**
Please take a look at  home task 1

### CODE REVIEW

**Review by PBaranovskyi — COMMENTED** [review]
good - добавь задачу с типами из хакерранка

**Comment by Tretykhlib** [comment]
added task from hackerrank

File: homeworks/src/ltretykhlib/module1/example/Hackerrank2.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        Scanner scan = new Scanner(System.in);
+
+        int my = scan.nextInt();
+        System.out.println(i + my);
+
+        double k = scan.nextDouble();
+        System.out.println(d + k);
+
+        String str;
+        str= scan.nextLine();
```
Comment (PBaranovskyi) [inline homeworks/src/ltretykhlib/module1/example/Hackerrank2.java]: why two times nextLine?

File: homeworks/src/ltretykhlib/module1/example/PrimitiveDataTypes.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+/**
+ * Created by Tretykhlib on 06.04.2017.
+ */
+public class PrimitiveDataTypes {
+    public static void main(String[] args) {
+        int i = 10000;
+        short s = 1000;
+        double d = 101.2;
+        float f = 100.1f;
```
Comment (PBaranovskyi) [inline homeworks/src/ltretykhlib/module1/example/PrimitiveDataTypes.java]: use please Upper case literal suffixes

File: homeworks/src/ltretykhlib/module1/example/ArithmeticOperators.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ */
+public class ArithmeticOperators {
+    public static void main(String[] args) {
+
+        int a = 67687;
+        int b = 15703;
+
+        int sum = a + b;
+        int subs = a-b;
+        int div=a/b;
```
Comment (PBaranovskyi) [inline homeworks/src/ltretykhlib/module1/example/ArithmeticOperators.java]: Format all your code please with ctrl+alt +L

**Comment by PBaranovskyi** [comment]
Format all your code please with ctrl+alt +L . And add tasks from hacker rank please

**Comment by Tretykhlib** [comment]
Please merge with master

**Review by PBaranovskyi — CHANGES_REQUESTED** [review]

File: homeworks/src/ltretykhlib/module1/example/PrimitiveDataTypes.java (hunk 2 above)
Comment (Tretykhlib) [reply homeworks/src/ltretykhlib/module1/example/PrimitiveDataTypes.java]: change applied

File: homeworks/src/ltretykhlib/module1/example/ArithmeticOperators.java (hunk 3 above)
Comment (Tretykhlib) [reply homeworks/src/ltretykhlib/module1/example/ArithmeticOperators.java]: change applied

**Pull request merged**